content/livesync/models.textile

```diff
@@ -209,42 +285,25 @@ INSERT INTO outbox (mutation_id, channel, name, ...) VALUES ('my-mutation-id', '
 COMMIT;
 ```
 
-h4(#rejections). Rejections and timeouts
+h2(#subscribe). Subscribe to changes in the model
 
-The SDK automatically rewinds the optimistic event and rejects the associated confirmation promise if the corresponding confirmed change event is not received from the backend within a specified timeframe. You can configure this timeout period using the @optimisticEventOptions.timeout@ "@ClientOption@":/livesync/models#options.
-
-You can also broadcast a rejection event from your backend in order to explicitly reject a given optimistic update. This is achieved by setting the rejected flag to true in the "outbox":/livesync/connector/database#outbox record:
-
-You can reject optimistic updates by broadcasting a rejection event from your backend. Set the rejected flag to true in the "outbox":/livesync/connector/database#outbox record:
-
-```[sql]
-BEGIN;
--- mutate your data, e.g.:
-INSERT INTO comments (comment) VALUES ('New comment!');
--- write change event to outbox, e.g.:
-INSERT INTO outbox (mutation_id, channel, name, rejected, ...) VALUES ('my-mutation-id', 'posts:123', 'addComment', true, ...);
-COMMIT;
-```
-
-h2(#subscribe). Subscribe to changes
-
-To receive realtime notifications when there are changes in the state, you can subscribe to changes to the changes. 
+To receive realtime notifications when there are changes in the state, you can subscribe to changes to the changes.
 
 The following example subscribes to model state changes:
 
 ```[javascript]
 model.subscribe((err, post) => { /* model state updated! */ });
 ```
 
-Subscriptions operate on an optimistic model by default. The callback is triggered whenever the optimistic state changes or confirmed changes occur. 
+Subscriptions operate on an optimistic model by default. The callback is triggered whenever the optimistic state changes or confirmed changes occur.
 
 To respond solely to confirmed changes, set the @optimistic@ option to @false@ in the @subscribe()@:
 
 ```[javascript]
 model.subscribe((err, post) => { /* ... */ }, { optimistic: false });
 ```
 
-h2(#lifecycle). Model lifecycle
+h2(#model-lifecycle). Model lifecycle
 
 A model instance can be in one of the following lifecycle states:
 
```

Comment:
Not sure I understand "the model is attached to the channel and processing realtime." for `ready`

Reply:
Perhaps it is realtime updates?